packages/react-resizable-panels/package.json

```diff
@@ -17,12 +17,32 @@
         "import": "./dist/react-resizable-panels.cjs.mjs",
         "default": "./dist/react-resizable-panels.cjs.js"
       },
-      "development": {
-        "browser": {
+      "browser": {
+        "development": {
+          "worker": {
+            "module": "./dist/react-resizable-panels.development.worker.esm.js",
+            "import": "./dist/react-resizable-panels.development.worker.cjs.mjs",
+            "default": "./dist/react-resizable-panels.development.worker.cjs.js"
+          },
           "module": "./dist/react-resizable-panels.browser.development.esm.js",
           "import": "./dist/react-resizable-panels.browser.development.cjs.mjs",
           "default": "./dist/react-resizable-panels.browser.development.cjs.js"
         },
+        "worker": {
```

Comment:
This one is a little bit suboptimal. It's not incorrect - it's just that perhaps this could be somehow avoided to minimize the `exports` map? cc @emmatown 

The thing that is suboptimal is that `worker` is both nested here in `browser` and it's also used as a standalone "top-level" condition. It could be simply the top-level condition if only it would came **before** the `browser` condition.